glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
On Wed, Aug 26, 2015 at 02:09:43PM -0700, W. Trevor King wrote:
> Wed, Aug 26, 2015 at 07:31:50AM -0700, Brandon Philips:
> > a single bundle is intended to launch a single process at this
> > point.
> 
> Is there a reason to avoid allowing multiple application specs
> inside a bundle?

To tie this back into the current spec phrasing, I think:

  # Standard operations (lifecycle)
  
  ## Create
  
  Creates the container: file system, namespaces, cgroups, capabilities.
  
  ## Start (process)
  
  Runs a process in a container. Can be invoked several times.

is fairly clear in its desire for a separate container-creation
operation after which you can run several application-launching start
operations.  That wording landed in 77d44b10 (Update runtime.md,
2015-06-16) and is still current with today's 6a5a4454.

That doesn't mean that there portions of the spec that are phrased to
sound like there will only be a single application per bundle
(although I don't have one that I can quote here), but I think it
makes my plural in this PR plausible enough that we can merge it as it
stands ;).  We can always file a follow-up PR removing my plural and
rephrasing the lifecycle descriptions if/when we decide that we need a
single application per bundle.